extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -61,160 +65,201 @@ public function execute( $subpage ) {
 		/**
 		 * initial output
 		 */
-		$wgOut->setPageTitle( wfMsg('wikifactorypagetitle') );
-		$wgOut->setRobotpolicy( 'noindex,nofollow' );
+		$wgOut->setPageTitle( wfMsg( 'wikifactorypagetitle' ) );
+		$wgOut->setRobotPolicy( 'noindex,nofollow' );
 		$wgOut->setArticleRelated( false );
 
-		if ( in_array( strtolower($subpage), array( "metrics", "metrics/main", "metrics/monthly", "metrics/daily" ) ) ) {
+		if ( in_array( strtolower( $subpage ), self::METRIC_PATHS ) ) {
 			$oAWCMetrics = new WikiMetrics();
 			$oAWCMetrics->show( $subpage );
-		}
-		elseif ( strpos( $subpage, "short.stats" ) === 0 ) {
+		} elseif ( strpos( $subpage, "short.stats" ) === 0 ) {
 			$subpageOptions = explode( '/', $subpage );
 			$lang = isset( $subpageOptions[1] ) ? $subpageOptions[1] : null;
 			$wgOut->addHTML( $this->shortStats( $lang ) );
-		}
-		elseif ( strpos( $subpage, "long.stats" ) === 0 ) {
+		} elseif ( strpos( $subpage, "long.stats" ) === 0 ) {
 			$subpageOptions = explode( '/', $subpage );
 			$lang = isset( $subpageOptions[1] ) ? $subpageOptions[1] : null;
 			$wgOut->addHTML( $this->longStats( $lang ) );
-		}
-		elseif ( strtolower($subpage) === "add.variable" ) {
-			$varOverrides = array();
-			$wgOut->addHTML( $this->doAddVariableForm($varOverrides) ); // process the post (if relevant).
-			$wgOut->addHTML( $this->addVariableForm($varOverrides) ); // display the form
-		}
-		else {
+		} elseif ( strtolower( $subpage ) === "add.variable" ) {
+			$varOverrides = [];
+			$wgOut->addHTML( $this->doAddVariableForm( $varOverrides ) ); // process the post (if relevant).
+			$wgOut->addHTML( $this->addVariableForm( $varOverrides ) ); // display the form
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)